build_system/clerk_driver.ml

```diff
@@ -700,16 +709,37 @@ let add_test_builds
          else collect_in_file ctx file_or_folder curr_ninja reset_test_outputs)
        ctx
 
+let makeflags_to_ninja_flags (makeflags : string option) =
+  match makeflags with
+  | None -> ""
+  | Some makeflags ->
+      let ignore_rex = Re.Pcre.regexp "\\bi\\b" in
+      let has_ignore = Re.Pcre.pmatch ~rex:ignore_rex makeflags in
+      let jobs_rex = Re.Pcre.regexp "-j(\\d)" in
+      let number_of_jobs =
+        try
+          int_of_string
+            (Re.Pcre.get_substring (Re.Pcre.exec ~rex:jobs_rex makeflags) 1)
```

Comment:
Nitpicking here, but why use `Pcre` ?
It's generally considered much more legible to use the building functions of `Re` (although the regexps here remain pretty trivial).

```suggestion
      let ignore_rex = Re.(compile @@ word (char 'i')) in
      let has_ignore = Re.execp ignore_rex makeflags in
      let jobs_rex = Re.(compile @@ seq [str "-j"; group (rep digit)]) in
      let number_of_jobs =
        try
          int_of_string
            (Re.get (Re.exec jobs_rex makeflags) 1)
```